nbs/07_quickstart_for_web_devs.ipynb

```diff
@@ -159,7 +161,7 @@
    "source": [
     "## Routing\n",
     "\n",
-    "Friendly URLs are intuitive and helpful to users, and easier to maintain for coders. Users are more likely to use your project if they can remember URLs without much effort. Fortunately, FastHTML uses the Python community's common pattern for specifying URLs.\n",
+    "FastHTML uses the Python community's friendly decorator pattern for specifying URLs:\n",
```

Comment:
I shortened this, but it potentially loses your intent here? By "friendly URLs" do you mean that decorating functions with `@rt` is friendly and intuitive compared with, say, Django's separate urls.py that you have to cross-reference in another file?

Reply:
@jph00 raised the same issue. I was originally talking about easy-to-understand URLs but that's dating myself to 2010. I think your text is better for 2024.